otter/test/test_supervisor.py

```diff
@@ -56,12 +57,13 @@ def setUp(self):
         self.group.uuid = 'group-id'
         self.region = "ORD"
 
-        self.auth_token = 'auth-token'
+        self.auth_token = ('auth-token1', 'auth-token2', 'auth-token3')
+        self._auth_token = list(self.auth_token)
```

Comment:
I suggest naming these variables "self.auth_tokens" and "self.auth_token_queue" or something like that